mm/ktsan/supp.c

```diff
@@ -0,0 +1,55 @@
+#include "ktsan.h"
+
+#include <linux/kernel.h>
+#include <linux/kallsyms.h>
+
+#ifndef CONFIG_KALLSYMS
+#error "KTSAN suppressions require CONFIG_KALLSYMS"
+#endif
+
+struct kt_supp_s {
+	const char *func;
+	unsigned long start;
+	unsigned long end;
+	unsigned hits;
+};
+typedef struct kt_supp_s kt_supp_t;
+
+static kt_supp_t suppressions[] = {
+	{"generic_fillattr"},
+	{"atime_needs_update"},
+	{"tcp_poll"},
```

Comment:
Maybe it makes sense to add comments explaining why each of these races is suppressed.